.drone.yml

```diff
@@ -45,91 +48,99 @@ pipeline:
       - php occ app:enable files_antivirus
       - php occ app:enable testing
       - php occ app:list
-      - php occ config:system:set trusted_domains 1 --value=server
+      - php occ config:system:set trusted_domains 1 --value=owncloud
       - php occ log:manage --level 0
+    when:
+      matrix:
+        NEED_INSTALL_APP: true
+
+  configure-app:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/
+      - php occ config:app:set --value "clamav" files_antivirus av_host
+      - php occ config:app:set --value "daemon" files_antivirus av_mode
+      - php occ config:app:set --value "3310" files_antivirus av_port
+    when:
+      matrix:
+        TEST_SUITE: api-acceptance
+
+  fix-permissions:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - chown www-data /var/www/owncloud -R
+      - chmod 777 /var/www/owncloud/tests/acceptance/filesForUpload -R
+      - chmod +x /var/www/owncloud/tests/acceptance/run.sh
+    when:
+      matrix:
+        NEED_SERVER: true
 
   owncloud-log:
     image: owncloud/ubuntu:16.04
     detach: true
     pull: true
     commands:
     - tail -f /var/www/owncloud/data/owncloud.log
+    when:
+      matrix:
+        NEED_SERVER: true
 
-  code-compliance-check:
+  owncloud-coding-standard:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
       # currently failing
       #- make test-php-codecheck
-      - make test-php-lint
       - make test-php-style
     when:
       matrix:
-        TEST_SUITE: phpunit
+        TEST_SUITE: owncloud-coding-standard
 
-  phpunit:
+  phpunit-tests:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
+    environment:
+      - PHP_VERSION=${PHP_VERSION}
```

Comment:
What is the reason for `PHP_VERSION` ? Where do we use this variable and why is it important?

Reply:
It is not important. I will remove it and make a note to myself to remove it everywhere else.

It was copied from ``.drone.yml`` in password_policy, files_texteditor and probably a few other apps.
This is where we need a "gold" standard to copy-paste from. Because we keep having to think over-and-over if the code is optimal.